plugin/trino-loki/src/main/java/io/trino/loki/LokiRecordCursor.java

```diff
@@ -178,39 +175,40 @@ public long getLong(int field)
     {
         Type type = getType(field);
         if (type.equals(LokiMetadata.TIMESTAMP_COLUMN_TYPE)) {
-            Long nanos = (Long) requireNonNull(getEntryValue(field));
-            // render with the fixed offset of the Trino server
-            int offsetMinutes = Instant.ofEpochMilli(nanos / 1000).atZone(ZoneId.systemDefault()).getOffset().getTotalSeconds() / 60;
-            return packTimeWithTimeZone(nanos, offsetMinutes);
+            Long ts = (Long) requireNonNull(getFieldValue(field));
```

Comment:
Ideally `getFieldValue` would return an `Instant`. However, I didn't get around to deserialize the different timestamps with Jackson :shrug: